site2/docs/admin-api-namespaces.md

```diff
@@ -178,11 +178,11 @@ admin.namespaces().getNamespaceReplicationClusters(namespace)
 
 #### set backlog quota policies
 
-Backlog quota helps broker to restrict bandwidth/storage of a namespace once it reach certain threshold limit . Admin can set this limit and one of the following action after the limit is reached.
+Backlog quota helps the broker to restrict bandwidth/storage of a namespace once it reach a certain threshold limit. Admin can set this limit and one of the following actions after the limit is reached.
 
   1.  producer_request_hold: broker will hold and not persist produce request payload
 
-  2.  producer_exception: broker will disconnects with client by giving exception
+  2.  producer_exception: broker will disconnect with the client by giving an exception
```

Comment:
could you please refine all of them as present tense (rather than future tense)? Thank you.
Usually, we use present tense in technical writing. For details, refer to https://developers.google.com/style/tense